src/test/java/uk/gov/hmcts/cmc/claimstore/rules/ClaimantRepaymentPlanRuleTest.java

```diff
@@ -0,0 +1,172 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.hamcrest.core.StringContains;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ClaimantInvalidRepaymentPlanException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.response.FullAdmissionResponse;
+import uk.gov.hmcts.cmc.domain.models.response.PartAdmissionResponse;
+import uk.gov.hmcts.cmc.domain.models.response.PaymentIntention;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleRepaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.response.SamplePaymentIntention;
+
+import java.time.LocalDate;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClaimantRepaymentPlanRuleTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    private ClaimantRepaymentPlanRule claimantRepaymentPlanRule = new ClaimantRepaymentPlanRule();
+
+    private final PaymentIntention installmentPaymentIntentionBeforeOneMonth = SamplePaymentIntention.builder()
```

Comment:
`instalment` is misspelt in several places. Quick search/replace will fix that 👍